.github/workflows/build.yml

```
@@ -1,7 +1,10 @@
 name: CI
 
-on: 
+on:
   push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

```suggestion
  [push, pull_request]:
    branches: 
      - master
```

Syntax taken from [github docs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows). We can put the duplicates in a list and put `master` by itself

---

if we don't want to run CI twice, we can do this
```yaml
on:
  push:
    branches-ignore:
      - master
  pull_request:
    branches:
      - master
```